### DISCUSSION TITLE
[Import ASCII data] Hello, RGPR team.

### DISCUSSION THREAD

**Discussion by Changwan-planet** [emanuelhuber/RGPR#72] [General]
Dear RGPR developers,

Hello, Thank you for opening this nice program. 
I have a question about importing the text data in your program.

I read the content in your website below:

**Either 3-column format (x, t, amplitude) or matrix-format (without header/rownames). readGPR() should be able to detect the format as well as the separator.**

But I have a struggle to read it. 
For example, my text file (*.txt) includes the contents below:

1  1 4.1659247330347294E-004
2  1 1.6948349871485300E-004
3  1 1.4013759846954617E-004
4  1 3.2717632059802050E-004
5  1 1.6493087284119581E-004
6  1 2.5842575442402087E-004
7  1 1.6449641495455434E-004
8  1 1.5897951344909575E-004
9  1 1.6084049815756457E-005
10  1 9.2603714914214343E-005

I got this error:
```
"Error in [.data.frame(X, , pos[1]): undefined columns selected."
```

Even though I add comma(,) or tab between elements, I had the same error. 

Would you mind sharing examples of text (ASCII) data reading in RGPR?

Best, 
Changwan Sun

**Comment by emanuelhuber** [comment]
Hello @Changwan-planet 

Thank you for opening this discussion.
I have not check what could raise the error you get, but I think that the problem is the "E-004".
Write you .txt file with "normal" numbers, e.g., 0.00041659247330347294 (instead of 4.1659247330347294E-04).

Best,

Emanuel

**Comment by Changwan-planet** [comment]
Thank you for your reply. I will try that!
